src/sql-parser/src/ast/defs/value.rs

```diff
@@ -59,6 +62,21 @@ pub enum Value {
     Null,
 }
 
+impl Value {
```

Comment:
I think this block still needs to be removed.

Reply:
Thanks, totally slipped my mind. Just deleted it.